frontend/src/exams/add-exam-modal.vue

```diff
@@ -285,11 +290,13 @@
         let { setup } = this.addExamModal
         this.unSubmitted = false
         this.submitMsg = ''
+        console.log("SETUP", setup)
```

Comment:
Not gonna fail this PR for this, but can you remove this console log in your next jest push?

Reply:
fixed.